10/umbraco-workflow/README.md

```diff
@@ -1,15 +1,36 @@
-# What is Umbraco Workflow?
+---
+description: Documentation on how to work with Umbraco Workflow in just a few steps
+layout: landing
```

Comment:
Doesn't "landing" mean, that the article hides the left- and right-hand side navigations? 🤔 

Also, I believe the description should be in quotation marks.

Reply:
These tags/values are generated by GitBook. I only selected Page Layout as Landing.